.github/workflows/call-buildimage.yml

```diff
@@ -9,7 +9,7 @@ on:
 
 jobs:
   build-publish-image:
-    uses: felleslosninger/eid-github-workflows/.github/workflows/eid-spring-boot-build-publish-image-generic.yml@main
+    uses: felleslosninger/eid-github-workflows/.github/workflows/spring-boot-build-publish-image.yml@main
```

Comment:
-config?

Reply:
nei, her er det brukt 2 workflow jobbar i staden for 1.